UdpDebugLog/src/SphereUdpLogSender/main.c

```diff
@@ -0,0 +1,48 @@
+﻿/* Copyright (c) Microsoft Corporation. All rights reserved.
+   Licensed under the MIT License. */
+
+#include <stdbool.h>
+#include <errno.h>
+#include <string.h>
+#include <time.h>
+
+// See CMakeLists.txt to control whether USE_SOCKET_LOG is defined or not.
+// If this is defined, then the Log_Debug call is sent via a UDP socket broadcast as well as printed on stderr (which goes to the PC when the app is started under debugger, like the standard Log_Debug).
+// If this is not defined, then the standard Log_Debug is used.
+
+// Note: This code uses a socket for output, this could be replaced with UART or other interface
+
+#ifdef USE_SOCKET_LOG
```

Comment:
I'm not sure this ifdef or header is needed.  applibs/log.h will define the signature, which you're matching.  The linker will prefer your version because it's being directly linked.

Reply:
the header can be used as a replacement for applibs, or could be removed.